core/monitorstorage/storage.go

```diff
@@ -52,6 +52,7 @@ type Result struct {
 }
 
 // Interface is the interface that stores and retrieves monitoring results.
+// TODO(gbelvin): make multi-tennent.
```

Comment:
tenant.  Tennent is a type of high alcohol lager favoured in Scotland :)

Reply:
Thanks.